src/RegisterPage.h

```diff
@@ -57,10 +57,13 @@ private slots:
         void onBackButtonClicked();
         void onRegisterButtonClicked();
 
-        // Display registration specific errors to the user.
-        void registerError(const QString &msg);
+        // function for showing different errors
```

Comment:
I don't think changing this comment makes anything clearer. I'd prefer to keep the old one.

Reply:
Agree :)